docs/modules/ROOT/pages/index.adoc

```diff
@@ -23,3 +25,7 @@ See xref:migration_guide.adoc[Devfile v2 Migration Guide].
 == Devfile v2.0 Examples
 
 See https://github.com/devfile/api/tree/master/samples/devfiles[Devfile Samples].
+
+== Additional Resources
+
+For a visual and audible demonstration of devfiles, see the YouTube video, https://youtu.be/vnsLwtRz--w[Customize your own online IDE with a devifle].
```

Comment:
That video is based on devfile 1.0. Although some of the concepts that it shows still apply, the sample devfile that it uses don't match devfile 20 so I wonder if we should include that as part of this page. It may introduce confusion if the user tries to follow what shows on the video.